my-app/src/app/core/services/loader.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoaderService {
+  loaderActive$ = new Subject<boolean>();
```

Comment:
it might be useful to use ReplaySubject here, so component will get the current state no matter when it was created.